lib/spack/spack/build_systems/cmake.py

```diff
@@ -83,14 +83,21 @@ def build_type(self):
         return 'RelWithDebInfo'
 
     @property
-    def root_cmakelists_dir(self):
-        """Returns the location of the root CMakeLists.txt
+    def cmakelists_directory(self):
+        """The relative path to the directory containing CMakeLists.txt
```

Comment:
I am not sure I find this name clearer than the previous one... I named the function `root_cmakelists_dir` because you may have more than one `CMakeLists.txt` file in your project, residing in different directories. Usually you want to point to the one in the root which will contain the `project` command, hence the name used before.